src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="IMac image"
+        class="card__image"
+      >
+
+      <section class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </section>
+
+      <section class="card__code">
+        Product code: 195434
+      </section>
+
+      <section class="card__rating">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <div class="card__reviews">
+          Reviews: 5
+        </div>
+      </section>
+
+      <section class="card__price">
+        <div class="card__price-label">
+          Price:
+        </div>
+
+        <div class="card__price-amount">
+          $2,199
+        </div>
+      </section>
+
+      <button class="card__buy card__buy--hover">
+        <a
+          href="#buy"
+          class="card__link card__link--hover"
+          data-qa="hover"
+        >
+          BUY
```

Comment:
Let's transform the text with css property, not directly in HTML

Reply:
fixed

Reply:
fixed